lib/Chat/Parser/UserMention.php

```diff
@@ -116,8 +116,8 @@ public function parseMessage(Message $chatMessage): void {
 
 			if ($mention['type'] === 'call') {
 				$userId = '';
-				if ($chatMessage->getParticipant()->getAttendee()->getActorType() === Attendee::ACTOR_USERS) {
-					$userId = $chatMessage->getParticipant()->getAttendee()->getActorId();
+				if ($chatMessage->getParticipant()?->getAttendee()->getActorType() === Attendee::ACTOR_USERS) {
+					$userId = $chatMessage->getParticipant()?->getAttendee()->getActorId();
```

Comment:
`getParticipant()` can't return `null` on line `120` unless `Attendee::ACTOR_USERS` is null.
I think it can also be written like this: 
```php
$userId = $chatMessage->getParticipant()?->getAttendee()->getActorType() === Attendee::ACTOR_USERS
    ? $chatMessage->getParticipant()->getAttendee()->getActorType()
    : ''
```

Reply:
For reasons I don't understand, EA things otherwise:
![grafik](https://github.com/nextcloud/spreed/assets/213943/d5f2fbfb-d685-4b14-8733-7b88b1827ac0)

Reply:
I guess we are smarter than him :smile:

Reply:
Well I guess EA can not know, if getParticipant() would always return the same thing. It could pop items from a list, etc.

Reply:
```php
$userId = $chatMessage->getParticipant()?->getAttendee()->getActorType() === Attendee::ACTOR_USERS
    ? $chatMessage->getParticipant()?->getAttendee()->getActorType()
    : ''
```
We could write it with ternary anyway